ci-operator/step-registry/openstack/provision/bastionproxy/openstack-provision-bastionproxy-commands.sh

```diff
@@ -135,13 +139,11 @@ echo ${SQUID_AUTH}>${SHARED_DIR}/SQUID_AUTH
 MACHINES_GATEWAY_IP=""
 SQUID_IP=$bastion_fip
 if [[ "${CONFIG_TYPE}" == "proxy" ]]; then
-  # Right now we assume that the bastion will be connected to one machines network via a port.
-  # This command will have to be revisited if we want more ports on this machine.
-  PROXY_INTERFACE="$(openstack port list --network $MACHINES_NET_ID --server "$server_id" \
-    -c fixed_ips -f value |cut -d':' -f3 |cut -f1 -d '}' |sed -e "s/'//" -e "s/'$//")"
   SQUID_IP=$PROXY_INTERFACE
-  echo ${PROXY_INTERFACE}>${SHARED_DIR}/PROXY_INTERFACE
```

Comment:
You still need to persist PROXY_INTERFACE in SHARED_DIR otherwise the `openstack-conf-installconfig` step skips proxy configuration [as shown here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/35545/rehearse-35545-periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.13-e2e-openstack-proxy/1616493684646219776/artifacts/e2e-openstack-proxy/openstack-conf-installconfig/artifacts/install-config.yaml).

Reply:
Done.